src/features/rewards/disabledBox/style.ts

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this file,
+* You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+import styled from 'styled-components'
+import Card from '../../../components/layout/card'
+
+interface StyleProps {
+  padding?: boolean
+}
+
+export const StyledWrapper = styled<{}, 'div'>('div')`
+  display: block;
+  width: 100%;
+  margin-bottom: 28px;
+`
+
+export const StyledCard = styled(Card)`
+  font-family: Poppins, sans-serif;
+  border-bottom-left-radius: 6px;
```

Comment:
why is this needed as `Card` already has border radius se 6px for all sides?

Reply:
Rules removed